src/c/sea-lines.c

```diff
@@ -1,27 +1,24 @@
 #include <pebble.h>
 #include "utils.h"
+#include "config.h"
 
 #define DEBUG_TIME (false)
 #define BUFFER_LEN (10)
-#define COL_BG1   (GColorOxfordBlue)
-#define COL_BG2   (GColorLiberty)
-#define COL_HOUR  (GColorCeleste)
-#define COL_MIN   (GColorRajah)
-#define COL_BEIGE (GColorPastelYellow)
 
 static Window* s_window;
 static Layer* s_layer;
 static char s_buffer[BUFFER_LEN];
+static AppConfig s_config;
 
 static void draw_bg(GContext* ctx, GRect bounds, GPoint center, int vcr) {
   int width = 4;
   int i = 0;
   int to_corner = 1414 * max(bounds.size.h, bounds.size.w) / 1000;
   for (int radius = to_corner; radius > width; radius -= width) {
     if (i++ % 2 == 0) {
-      graphics_context_set_fill_color(ctx, COL_BG1);
+      graphics_context_set_fill_color(ctx, (&s_config)->bg1);
```

Comment:
nit-pick: Why `(&s_config)->bg1`? Isn't that a more convoluted way of saying `s_config.bg1`?

Reply:
You are right! I was clearly not thinking right yesterday